tests/e2e/util.go

```diff
@@ -3209,15 +3296,28 @@ func GetPodSpecByUserID(ns string, nodeSelector map[string]string, pvclaims []*v
 
 // writeDataOnFileFromPod writes specified data from given Pod at the given.
 func writeDataOnFileFromPod(namespace string, podName string, filePath string, data string) {
-	_, err := framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
-		podName, "--", "/bin/sh", "-c", fmt.Sprintf(" echo %s >  %s ", data, filePath))
+	var err error
+	if windowsEnv {
+		_, err = framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
+			podName, "--", "Powershell.exe", "-Command", fmt.Sprintf(" echo '%s' >  %s ", data, filePath))
+	} else {
+		_, err = framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
+			podName, "--", "/bin/sh", "-c", fmt.Sprintf(" echo %s >  %s ", data, filePath))
+	}
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 }
 
 // readFileFromPod read data from given Pod and the given file.
 func readFileFromPod(namespace string, podName string, filePath string) string {
-	output, err := framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
-		podName, "--", "/bin/sh", "-c", fmt.Sprintf("less  %s", filePath))
+	var err error
+	var output string
+	if windowsEnv {
+		output, err = framework.RunKubectl(namespace, "exec", fmt.Sprintf("--namespace=%s", namespace),
+			podName, "--", "Powershell.exe", "-Command", fmt.Sprintf("Get-Content %s", filePath))
```

Comment:
if we are landing into powershell when we do exec, we don't have to add any new code